pkg/daemon/ceph/client/upgrade.go

```diff
@@ -180,6 +180,7 @@ func okToStopDaemon(context *clusterd.Context, clusterInfo *ClusterInfo, deploym
 		args := []string{daemonType, "ok-to-stop", daemonName}
 		buf, err := NewCephCommand(context, clusterInfo, args).Run()
 		if err != nil {
+			logger.Infof("deployment %s cannot be stopped, error:%v", deployment, err)
 			return errors.Wrapf(err, "deployment %s cannot be stopped", deployment)
```

Comment:
The details of the failure are found in the `buf` return value above. Let's just include that in the error message that is returned. We don't want to add a new log statement here because it retries a number of times while waiting for success, and it would potentially print the same error many times instead of only at the end of the retries if we just update this error message.
```suggestion
			return errors.Wrapf(err, "deployment %s cannot be stopped. %s", deployment, string(buf))
```

Reply:
Done.
Thank you, Travis, for your comment. I’d like to reproduce the error in my local environment (Minikube). Could you suggest a method or approach to achieve this?

Reply:
An OSD is not ok-to-stop if the `min_size` of a pool cannot be satisfied if the OSD goes down. So if you have 3 OSDs and a replica-3 pool (with default min_size=2), one OSD can go down, but two OSDs cannot go down. You could scale down one OSD to simulate one OSD being down, then Rook should hit this code path that another OSD is not ok-to-stop during upgrade.

Reply:
@travisn On my setup, I am working with Minikube with 1 node and 1 OSD (replica 1). I have not tested with 3 nodes and 3 OSDs. Is it necessary to test this configuration before merging the PR? If yes, I will explore how to deploy such a setup locally on my machine.

Reply:
It could be tested with two OSDs in minikube, with replica 2 and `failureDomain: osd` in the pool. But the change for the error message is simple enough I'm ok to merge without testing it.